docs/source/author/chapters.rst

```diff
@@ -52,11 +50,9 @@ Configuration Files
 ==============================================================================
 
 This section details configuration files within the ``content/structure`` directory.
-These files are in YAML format. If you are not familiar with YAML, see
-:doc:`understanding_configuration_files`.
+These files are in YAML format. If you are not familiar with YAML, see :doc:`understanding_configuration_files`.
 
-The diagram below shows an example of YAML file locations for the
-``content/structure/`` language directory, where:
+The diagram below shows an example of YAML file locations for the ``content/structure/`` language directory, where:
 
 - Blue is directories.
 - Red is YAML configuration files.
```

Comment:
There is no blue or red in the diagram, I think something went wrong that stopped the syntax from markdown files being rendered properly